src/test/java/org/isf/lab/data/LaboratoryHelper.java

```diff
@@ -35,41 +36,64 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
```

Comment:
Are these modules needed only for Lab tests or potentially for every test?

Reply:
I prefer to load them only if needed